### PR TITLE
Funnel trace description

### PR DESCRIPTION
**Pull request by nicolaskruchten**

### CODE REVIEW

**Comment by nicolaskruchten** [comment]
fixing

File: src/traces/funnel/index.js [hunk 1]
```diff
@@ -26,12 +26,12 @@ module.exports = {
     basePlotModule: require('../../plots/cartesian'),
     categories: ['bar-like', 'cartesian', 'svg', 'oriented', 'showLegend', 'zoomScale'],
     meta: {
-        description: [ // TODO: update description
-            'Draws funnel trace.',
-            '"Funnel charts are a type of chart, often used to represent stages in a sales process',
-            'and show the amount of potential revenue for each stage. This type of chart can also',
-            'be useful in identifying potential problem areas in an organization’s sales processes.',
-            'A funnel chart is similar to a stacked percent bar chart." (https://en.wikipedia.org/wiki/Funnel_chart)'
+        description: [
```
Comment (antoinerg) [inline src/traces/funnel/index.js]:
```
/home/circleci/plotly.js/src/traces/funnel/index.js
  29:23  error  Trailing spaces not allowed  no-trailing-spaces

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

**Comment by archmoj** [comment]
@nicolaskruchten Thanks.
:dancer:

**Comment by antoinerg** [comment]
:dancer:

**Pull request merged**